packages/components/checkbox-list/index.tsx

```diff
@@ -57,7 +57,9 @@ const CheckboxList = ( {
 					/* stylelint-disable */
 					width: Math.floor( Math.random() * 75 ) + 25 + '%',
 				} }
-			/>
+			>
+				&nbsp;
```

Comment:
I wonder if we should add a comment here that explains the purpose of `&nbsp;`. I'm thinking of how to prevent that someone, including me, accidentally removes `&nbsp;` in the future.

Reply:
Good idea, done!